include/ginkgo/c_api.h

```diff
@@ -0,0 +1,851 @@
+// SPDX-FileCopyrightText: 2017 - 2024 The Ginkgo authors
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Copyright (c) 2017-2023, the Ginkgo authors
+#ifndef C_API_H
+#define C_API_H
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+/* ----------------------------------------------------------------------
+ * Data type constants in GINKGO
+ * ---------------------------------------------------------------------- */
+enum _GKO_DATATYPE_CONST {
```

Comment:
I don't think this is used? Maybe remove it until it becomes necessary.

Reply:
Yes this was unused indeed, will remove it